apps/server/src/infra/rabbitmq/rpc-message-producer.ts

```diff
@@ -27,11 +27,15 @@ export abstract class RpcMessageProducer {
 	}
 
 	protected createRequest(event: string, payload: unknown) {
+		// expiration should be greater than timeout
+		const expiration = this.timeout > 0 ? this.timeout / 0.5 + this.timeout : undefined;
```

Comment:
`this.timeout / 0.5 + this.timeout` ist das selbe wie `3 * this.timeout`. Das wäre doch wesentlich einfacher zu lesen.

Reply:
@dyedwiper we were just testing you